limitador-server/kubernetes/limitador-deployment.yaml

```diff
@@ -16,7 +16,7 @@ spec:
     spec:
       containers:
         - name: limitador
-          image: quay.io/3scale/limitador:0.5.1
```

Comment:
there is way to break the egg-chicken issue. 

1. In this deployment object, we make reference to not-existing image, for example: `limitador:dev`
2. Build the image of the current workspace and tag it as `limitador:dev`.
3.  Before creating the deployment object in kind, load the image
```
kind load docker-image limitador:dev --name ${cluster-name}
```
4. Create the deployment object. The image being deployed will be the one recently built.

Maybe not for this PR